runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/ClueScrollConfig.java

```diff
@@ -40,4 +40,14 @@ default boolean displayHintArrows()
 	{
 		return true;
 	}
+
+	@ConfigItem(
+		keyName = "highlightStashUnits",
+		name = "Highlight STASH units",
+		description = "Configures whether or not to highlight STASH units for the emote clue scrolls"
+	)
+	default boolean highlightStashUnits()
```

Comment:
I dont think this needs to be configurable, we don't have config option for other highlights from clue scroll plugin too

Reply:
Removed